#!/bin/sh /usr/share/dpatch/dpatch-run
## 905-file-printf.dpatch by Martin Dorey <mdorey@bluearc.com>
##
## DP: Removing annoying warnings when files can't be opened (Closes: #481512).

@DPATCH@

diff -Naurp file-4.24.orig/src/magic.c file-4.24/src/magic.c
--- file-4.24.orig/src/magic.c	2008-05-04 14:12:49.000000000 +0000
+++ file-4.24/src/magic.c	2008-05-21 08:47:05.000000000 +0000
@@ -302,7 +302,6 @@ file_or_fd(struct magic_set *ms, const c
 			(void)strcat(strcpy(tmp, inname), ".exe");
 			if ((fd = open(tmp, flags)) < 0) {
 #endif
-				fprintf(stderr, "couldn't open file\n");
 				if (info_from_stat(ms, sb.st_mode) == -1)
 					goto done;
 				rv = 0;
